ivy/functional/ivy/general.py

```diff
@@ -1317,9 +1317,62 @@ def value_is_nan(
     ret
         Boolean as to whether the input value is a nan or not.
 
+    Examples
+    --------
+    >>> x = ivy.array([451])
+    >>> y = x.value_is_nan()
```

Comment:
sorry, I just realised, this should be `ivy.value_is_nan(x)` to showcase functional method